woocommerce/woocommerce
# [Accessible] Make tabs on product page accessible

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51924 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to a product page.
2. Scroll down to the tabs.
3. When focus is on a tab item:
     - Verify that pressing the Tab/Shift+Tab key on the keyboard moves the focus away from the tab list.
     - Verify it's possible to navigate through the tab items by pressing the arrows left/right keys on the keyboard.
     - Verify that the focus moves to the first tab item when the Home key is pressed on the keyboard.
     - Verify that the focus moves to the last tab item when the End key is pressed on the keyboard.
4. Turn on the screen reader.
5. Verify screen reader announces the selected tab item and its position (e.g. 2 of 3).
6. Verify it's possible to select a tab by pressing the Enter key on the keyboard.




### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
